samples/bluetooth/llpm/src/main.c

```diff
@@ -414,10 +412,24 @@ static void test_run(void)
 	}
 }
 
+void security_changed(struct bt_conn *conn, bt_security_t level,
+				 enum bt_security_err err)
+{
+	printk("Security changed: level %i, err: %i\n", level, err);
+
+	/*Start service discovery when link is encrypted*/
```

Comment:
can you add a check that `err` is non-zero and disconnect/assert if that's the case?

Reply:
done